### ISSUE TITLE
Icon component used in IconMessage is missing an aria-label attribute/incorrectly using an aria attribute and throwing a prop type warning

### ISSUE THREAD

**Issue by rdwoodring** [rdwoodring/seven-degrees-of-staind#131]

**Comment by rdwoodring** [comment]
Closed in 196292563c43f5b5fe3166837164c3c21b75d5a8 with incorrect commit message
--
